google/cloud/bigtable/instance_admin_client.cc

```diff
@@ -94,6 +94,15 @@ class DefaultInstanceAdminClient : public InstanceAdminClient {
     return impl_.Stub()->GetInstance(context, request, response);
   }
 
+  std::unique_ptr<grpc::ClientAsyncResponseReaderInterface<
```

Comment:
Please keep the functions in the `.cc` file in the same order they are found in the header file.  Much easier to find them later.